src/test/java/api/support/ApiRoot.java

```diff
@@ -37,6 +37,12 @@ public static URL items()
     return new URL(String.format("%s/items", inventory()));
   }
 
+  public static URL move()
```

Comment:
As I understand it, there will later be different kinds of move requests. Could it be worthwhile to refer to this one as specific to items?

```suggestion
  public static URL moveItems()
```